.ci/openshift-ci/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build and install kata-containers under <repository>/_out/build_install
+#
+
+set -e
+
+export GOPATH=${GOPATH:-${HOME}/go}
+cidir=$(dirname "$0")/..
+source "${cidir}/lib.sh"
+
+export kata_repo=$(git config --get remote.origin.url | sed -e 's#http\(s\)*://##')
+
+mkdir -p $cidir/../_out/build_install
+destdir=$(realpath $cidir/../_out/build_install)
+
+clone_tests_repo
+pushd "${tests_repo_dir}"
+# Resolve the kata-container repositories. It relies on branch and
+# kata_repo variables.
+export branch=master
```

Comment:
I still don't understand the tests scripts very well, with this you will be testing master and not the pr, is that correct ?

Reply:
yes @devimc . For kata-1.x I will run these scripts in a nightly job, so not dealing with PRs. I will need to adapt this to kata-2.x because in this case I plan to leverage the openshift CI to test PRs as well.